rimage/manifest.h

```diff
@@ -91,10 +91,14 @@ struct fw_image_manifest_v1_8 {
 extern struct fw_image_manifest_v1_8 apl_manifest;
 extern struct fw_image_manifest_v1_8 cnl_manifest;
 
+/*
+ * Firmware manifest header V1.5 used on SKL and KBL
+ */
 struct fw_image_manifest_v1_5 {
-	struct css_header_v1_5 header;
-	struct sof_man_fw_desc adsp_fw_bin_desc;
+	struct css_header_v1_5 css_header;
+	struct sof_man_fw_desc desc;
 } __attribute__((packed));
 
 extern struct fw_image_manifest_v1_5 skl_manifest;
+extern struct fw_image_manifest_v1_5 kbl_manifest;
 #endif
```

Comment:
do we really need platform-specific manifests?

Reply:
This is a good question, I compare man_apl.c and man_cnl.c and found only the load_offset is different. keep in one define will make the code simple. @plbossart Can this be refine after the patch? because the changes are related to old 1.8 apl and cnl code.

Reply:
@zhuyingjiang again, I'm not sure here until things are up and running - probably best adding another TODO item.

Reply:
OK